src/components/Advertiser/AdvertiserTable.vue

```diff
@@ -273,7 +273,7 @@ const errorStore = useErrorStore()
 const userStore = useUserStore()
 const selectedAdvertise = ref({})
 const filter = ref('')
-const selectedDataType = ref({ label: 'Active', value: 'active' })
+const selectedDataType = ref({ label: 'All', value: 'all' })
```

Comment:
We already discussed this point on another ticket but you changed it back to fetch all advertises again. Is necessary at this point? 🤔

<img width="1070" alt="Screenshot 2024-08-29 at 17 11 24" src="https://github.com/user-attachments/assets/ee24fbac-e1be-4ed8-bd8f-fde377547e54">

Reply:
I though fetching all the data and filtering it locally would be much faster.

Reply:
Why than we won't just fetch all the documents for the whole app at once and then just use it locally? 😅 

It's not only about how would be faster, even though fetching 10 documents is much faster than fetching 100 documents. It's also about the cost of the reads/writes. [Please take a look](https://firebase.google.com/docs/firestore/pricing) 

Probaly currently we shouldn't be bothered by the amount of requests made by 2-3 devs in development branch, but it could grow in the future (while it is still 2 million reads for a month for few users, imagine having 1000 users). And if we can make it be more optimized, well organized and scalable now it will only help us in the future. 

So please make it to NOT fetch all the advertisments at once, and query only the necessary once. To notify user for a pending ad, we could also query that and show some badge in the future if that's the case.

![Screenshot 2024-09-05 at 11 55 14](https://github.com/user-attachments/assets/85133f55-32a5-4d31-ba9a-7cf177b8adc7)

Reply:
I am working on it